etc/adminhtml/system/adyen_required_settings.xml

```diff
@@ -15,33 +15,35 @@
     <group id="adyen_required_settings" translate="label" type="text" sortOrder="20" showInDefault="1" showInWebsite="1"
            showInStore="1">
         <fieldset_css>adyen_required_config_settings</fieldset_css>
-
         <label><![CDATA[Required Settings]]></label>
+        <comment>
+            <![CDATA[<p>Use our <a target="_blank" href="https://docs.adyen.com/developers/plugins/magento-2/set-up-the-plugin-in-magento#step3configuretheplugininmagento">documentation</a> in case you have any doubt to fill in a specific information
```

Comment:
```suggestion
            <![CDATA[<p>Use our <a target="_blank" href="https://docs.adyen.com/plugins/magento-2/set-up-the-plugin-in-magento#step-3-configure-the-plugin-in-magento">documentation</a> in case you have any doubt to fill in a specific information
```